tests/utils/test_process_utils.py

```diff
@@ -0,0 +1,46 @@
+import os
+import tempfile
+import pytest
+
+from mlflow.utils.process import cache_return_value_per_process
+
+
+@pytest.mark.skipif(
+    os.name == "nt",
+    reason="Windows does not support fork",
+)
```

Comment:
Can we use Python's `multiprocessing` module instead of os.fork() in order to test Windows? https://docs.python.org/3/library/multiprocessing.html